spec/p2p/messages/state-sync.md

```diff
@@ -13,6 +13,7 @@ State sync has three distinct channels. The channel identifiers are listed below
 | SnapshotChannel   | 96     |
 | ChunkChannel      | 97     |
 | LightBlockChannel | 98     |
+| ParamsChannel     | 99     |
```

Comment:
does adding this channel break the protocol?

Reply:
No. If I'm running the new version and you're running the old version without this channel, then I simply don't send messages to you. That being said, if everyone is on the old version then I won't ever receive any response. In which case I'd have to revert back to using RPC state provider

Reply:
If the statesync provider is configurable at runtime, and both are supported, then people can switch between them as needed, so that should be fine.

Reply:
The text above this diff says 'state sync has three distinct channels'. Should now say four.

Reply:
Good spot 👍